stustapay/terminalserver/router/order.py

```diff
@@ -17,17 +17,15 @@ class NewOrderPayload(BaseModel):
     order: NewOrder
 
 
-@router.post("/order/create", summary="create and execute new order")
-async def create(
-    payload: NewOrderPayload,
+@router.get("/order", summary="list all orders")
+async def list_orders(
     token: str = Depends(get_auth_token),
     order_service: OrderService = Depends(get_order_service),
```

Comment:
we have a new mechanism for dependencies now, see the current master state (and maybe merge it already :)